CreateApiKey response now returns the base64 encoded credentials (#77351)
elastic/elasticsearch: x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

[source,console-result]
----
{
  "id": "VuaCfGcBCdbkQm-e5aOx",        <1>
  "name": "my-api-key",
  "expiration": 1544068612110,         <2>
  "api_key": "ui2lp2axTNmsyakw9tvNnw", <3>
  "encoded": "VnVhQ2ZHY0JDZGJrUW0tZTVhT3g6dWkybHAyYXhUTm1zeWFrdzl0dk5udw=="  <4>
}
----
// TESTRESPONSE[s/VuaCfGcBCdbkQm-e5aOx/$body.id/]
// TESTRESPONSE[s/1544068612110/$body.expiration/]
// TESTRESPONSE[s/ui2lp2axTNmsyakw9tvNnw/$body.api_key/]
// TESTRESPONSE[s/VnVhQ2ZHY0JDZGJrUW0tZTVhT3g6dWkybHAyYXhUTm1zeWFrdzl0dk5udw==/$body.encoded/]
<1> Unique `id` for this API key
<2> Optional expiration in milliseconds for this API key
<3> Generated API key

> <4> API key credentials which is the Base64-encoded string in UTF-8 format created
> by combining the `id` and `api_key` with a colon (`:`).

Reviewer: This terminology isn't really correct - it would be good to fix it up.
```suggestion
<4> API key credentials which is the Base64-encoding of the UTF-8 
representation of the `id` and `api_key` joined by a colon (`:`).
```

To use the generated API key, send a request with an `Authorization` header that

> contains an `ApiKey` prefix followed by the API key credentials.

Reviewer: ```suggestion
contains an `ApiKey` prefix followed by the API key credentials
(the `encoded` value from the response).
```
